examples/app/headless_renderer.rs

```diff
@@ -0,0 +1,444 @@
+//! This example illustrates how to make headless renderer
```

Comment:
This needs a lot more description of what this example actually does. It's not really explained anywhere what all the image copying is about or why it's done the way it is.